src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements Area {
```

Comment:
Тобі не потрібно імплементувати інтерфейси якщо вони імплементовані в клас, від якого ти наслідуєшся
```suggestion
public class Circle extends Figure {
```

Reply:
Дякую за clarifying :)